src/hooks/protections/received_event.cpp

```diff
@@ -100,6 +100,13 @@ namespace big
 		damageType = buffer->Read<uint8_t>(2);
 		weaponType = buffer->Read<uint32_t>(32);
 
+		if ("WEAPON_TRANQUILIZER"_J == weaponType)
+		{
+			if (auto plyr = g_player_service->get_by_id(player->m_player_id))
+    			g.reactions.break_game.process(plyr);

```

Comment:
Illformed code block.

Should be:

```
if (auto plyr = g_player_service->get_by_id(player->m_player_id))
{
    g.reactions.break_game.process(plyr);
}
```

Reply:
i was under the impression

if (auto plyr = g_player_service->get_by_id(player->m_player_id))
    			g.reactions.break_game.process(plyr);
    			
was how 1 lined if statements were supposed to be made. Changed it to yours now tho

Reply:
While it is valid syntax, and it has been seen around YimMenu before, it's not considered proper form. We're trying to move away from it.